.github/workflows/build.yml

```diff
@@ -413,32 +428,38 @@ jobs:
           python -m pip install pytest
 
       - name: Download artifacts
-        uses: actions/download-artifact@v1.0.0
+        uses: actions/download-artifact@v3
         with:
           name: wheels
+          path: wheels
 
       - name: get MininZinc path to cache
         id: get-mzn-cache-path
         run: |
-          echo "::set-output name=path::${{ env.minizinc_cache_path }}"  # expands variables
+          echo "path=${{ env.minizinc_cache_path }}" >> $GITHUB_OUTPUT
```

Comment:
Please do not remove comment, it is useful

Reply:
Pushed back command event though it is not expanding anything, just setting an env variable.